api/types/types.proto

```diff
@@ -302,6 +305,19 @@ message RDS {
     bool IAMAuth = 4 [ (gogoproto.jsontag) = "iam_auth" ];
 }
 
+// ElastiCache contains AWS ElastiCache Redis specific metadata.
+message ElastiCache {
```

Comment:
I'm not sure if we're going to support MemoryDB, but if so, would we need a new message type for that or we could just re-use this one?